framework/components/sequencecontroller.md

```diff
@@ -1,52 +1,115 @@
 # Sequence Controller
 
-## Overview
+## Motivation
 
-The Sequence controller will be mainly used to process LogicApp runs in order. The Sequence Controller exposes 3 actions:
+Some dependent external systems can't handle parallel message processing or requires a certain order of messages. Sending messages serially can be a solution, at the cost of a big performance impact. If the processing or sending of a single message is too big, serial is still not the solution.
```

Comment:
I don't quite understand this line

> If the processing or sending of a single message is too big, serial is still not the solution.


I think a better example to give is the following real-life example from a customer.
A downstream application cannot handle multiple calls at the same time for the same orderid, by using the sequencecontroller we are able to make sure that a single call per orderid is done (by using the orderid in the sequencename) and we do not have the use the concurrency option for the Logic App which would dramatically slow down the processing for all messages.

Reply:
Thx a lot for this, Pim!
Added 'a story from the trenches' here: https://github.com/invictus-integration/docs-ifa/pull/230/commits/6b80f615e9f6d332d86998533ad6953f7839c06b